pkg/params/params_test.go

```diff
@@ -513,3 +513,17 @@ func TestIsDefault(t *testing.T) {
 	p.Set("bar")
 	require.False(t, p.IsDefault())
 }
+
+func TestAlternativeKey(t *testing.T) {
```

Comment:
Do we have a behavior defined when we have multiple params with the same key? If yes we should also test where an `AlternativeKey` is the same as another params `Key`